### DISCUSSION TITLE
orchardcore and aspire dotnet ?

### DISCUSSION THREAD

**Discussion by giannik** [OrchardCMS/OrchardCore#17051] [Ideas]
I was looking at videos from the new dotnet aspire framework and wondering what use cases it can be used for in orchard core. My initial thoughts are around monitoring and observability. Does anyone else think Aspire can be utilized in orchard core as a module or is it too opinionated to work with orchard core ?

**Comment by hishamco** [comment]
Check this one https://github.com/OrchardCMS/OrchardCore/pull/14961
